addons/stock_account/static/src/fields/boolean_confirm.js

```diff
@@ -0,0 +1,59 @@
+/** @odoo-module **/
```

Comment:
@cth-odoo Do you think It could a standard feature ?
tl;dr: the same idea as `confirm=message` on a `<button>` but for a field. To be sure the value is actually changed only when the user click the confirm button of the modal. Thanks